src/App.tsx

```diff
@@ -1,8 +1,36 @@
+import { datadogRum } from "@datadog/browser-rum";
 import { SeoMetaTags } from "./components/SeoMetaTags/SeoMetaTags";
 import Theme from "./components/Theme";
 import { WithExtensionContext } from "./hooks/withExtensionContext";
+import { useEffect } from "react";
 
 function App() {
+  useEffect(() => {
+    const ddToken = import.meta.env.PUBLIC_DATADOG_TOKEN;
+    const ddId = import.meta.env.PUBLIC_DATADOG_ID;
+    const ddEnv = import.meta.env.PUBLIC_DATADOG_ENV;
+
+    if (ddToken && ddId && ddEnv) {
+      datadogRum.init({
+        applicationId: ddId,
+        clientToken: ddToken,
+        env: ddEnv,
+        site: "datadoghq.com",
+        service: "dc-extension-seo-meta-tags",
+        version: "1.1.0",
```

Comment:
Understandable, I wasn't sure about it either but noticed it's hardcoded in rich text as well so put it in. The `version` property is optional so we don't actually have to put anything for it if we don't want to, especially since the version number in `package.json` doesn't keep up with the version number from github releases

Reply:
yeah, probably be best to remove it.